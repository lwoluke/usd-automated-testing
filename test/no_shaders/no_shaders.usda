#usda 1.0
(
    doc = "A simple USD file."
)

def Xform "Root" {
    def Sphere "Sphere" {
    }
}
