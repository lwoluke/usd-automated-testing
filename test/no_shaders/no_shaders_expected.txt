Opened USD file Successfully.

[PASS] Validate Geometry: All geometry prims are valid.
[FAIL] Validate Shaders: No valid shaders found in the scene.
[PASS] Validate Layer Structure: Layer stack is valid.

Summary:
  Passed: 2
  Failed: 1

Some tests failed. Please review the USD file and address the failing tests.
